go/client/cmd_team.go

```diff
@@ -30,6 +30,7 @@ func NewCmdTeam(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.Command
 			newCmdTeamDelete(cl, g),
 			newCmdTeamAPI(cl, g),
 			newCmdTeamSettings(cl, g),
+			newCmdTeamProfileLoad(cl, g),
```

Comment:
This command should be hidden from normal users. Is it?

Reply:
the way you do that is to skip adding a `Usage` line.  Not the greatest, but that's what `cmd_riit.go` does, for instance.